### DISCUSSION TITLE
[백준 10156] 과자

### DISCUSSION THREAD

**Discussion by ghdcksgml1** [ghdcksgml1/Algorithm_Study#148] [1일 1알고리즘]
<img width="1156" alt="스크린샷 2022-03-12 오후 3 49 51" src="https://user-images.githubusercontent.com/79779676/158007497-5b120746-075b-404f-b4ba-6d6cda129001.png">

오랜만에 엄청 어려운 문제를 풀어보았다.

## 아이디어

아이디어가 잘 떠오르지 않았지만, 뭔가 사칙연산 알고리즘을 사용하면 될것같아 그렇게 구현해봤다.

## 시간복잡도

O(1)

## 소스코드

```c++

#include <iostream>

using namespace std;

int main(void){
    int K,N,M;
    
    cin>>K>>N>>M;
    int sum = K*N - M;
    if(sum <= 0) cout<<"0"<<'\n';
    else cout<<sum<<'\n';
}
```

**Comment by BanApp** [comment]
ㄷㄷ이걸 푸네 역시..
